src/common/styles/_typography.scss

```diff
@@ -0,0 +1,16 @@
+@font-face{
+    font-family:mainFont;
```

Comment:
It is better if we add the fonts with their original names. Also we need only Ubuntu-Regular.ttf file. You can rename the folder in utils from Ubuntu to fonts and delete any other files except Ubuntu-Regular.ttf
font-family: Ubuntu;